frontend/controller/actions/group.js

```diff
@@ -813,12 +807,12 @@ export default (sbp('sbp/selectors/register', {
             })
           } catch (e) {
             if (attempt > 3) {
-              console.error(`autoBanSenderOfMessage: max attempts reached. Error ${e.message} attempting to ban ${username}`, message, e)
+              console.error(`autoBanSenderOfMessage: max attempts reached. Error ${e.message} attempting to ban ${member}`, message, e)
             } else {
               const randDelay = randomIntFromRange(0, 1500)
-              console.warn(`autoBanSenderOfMessage: ${e.message} attempting to ban ${username}, retrying in ${randDelay} ms...`, e)
+              console.warn(`autoBanSenderOfMessage: ${e.message} attempting to ban ${member}, retrying in ${randDelay} ms...`, e)
```

Comment:
We still want to print out their username here as well.

I haven't finished reviewing the PR yet so I don't know, but if there isn't a getter to go from `identityContractID` => `username` then this is a great time/place to use it and add one

Reply:
There is (as you also found out in another later comment), it was `userUsername`, which now is `usernameFromID`.